src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+-------------------------------------------------------------
+-- Drop each of the schemas used by SDT and all their contents
+--------------------------------------------------------------
+--DROP SCHEMA IF EXISTS "SDT_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_BATCH_USER" CASCADE;
+--DROP SCHEMA IF EXISTS "SDT_OWNER" CASCADE;
+
+-------------------------------------------------------------
+-- Drop each of the users used by SD
+--------------------------------------------------------------
+---- non schema owners first
+--DROP USER IF EXISTS "SDT_USER";
+--DROP USER IF EXISTS "SDT_BATCH_USER";
+---- now schema owners
+--DROP USER IF EXISTS "SDT_OWNER";
+
+-------------------------------------------------------------
+-- Create the schemas used by SDT and all their contents
+--------------------------------------------------------------
+CREATE SCHEMA "SDT_OWNER";
+CREATE SCHEMA "SDT_USER";
+CREATE SCHEMA "SDT_BATCH_USER";
+
+------------------------------------------------
+-- Create each of the schemas used by SDT
+------------------------------------------------
+CREATE USER "SDT_OWNER";
+GRANT POSTGRES TO "SDT_OWNER";
+
+CREATE USER "SDT_USER";
+GRANT POSTGRES TO "SDT_USER";
+CREATE USER "SDT_BATCH_USER";
+GRANT POSTGRES TO "SDT_BATCH_USER";
+
+------------------------------------------------
+-- Create tables for SDT_OWNER
+------------------------------------------------
+CREATE TABLE "SDT_OWNER"."BULK_CUSTOMERS"
+   ("BULK_CUSTOMER_ID" NUMERIC NOT NULL, "SDT_CUSTOMER_ID" NUMERIC(8),
```

Comment:
Do the primary key columns need to be declared as "not null" within the table declarations?  Just wondering if the primary key declarations later on automatically set the columns to be not null.

Reply:
Unclear. No change.